lib/ansible/modules/cloud/alicloud/alicloud_image.py

```diff
@@ -350,7 +365,7 @@ def main():
                         images_ids.append(i.id)
                     module.fail_json(msg="There is too many images match name '{0}', "
                                          "please use image_id or a new image_name to specify a unique image."
-                                         "Matched images ids are: {1}".format(image_name, images_ids))
+                                             "Matched images ids are: {1}".format(image_name, images_ids))
         elif state == 'absent':
             images = ecs.get_all_images()

```

Comment:
I think line 369 can be remove. When state is absent and there is no matched image, we should prevent users to delete other images, like line 377.